src/functionalTest/resources/exception/sscs2_all_fields_entered.json

```diff
@@ -103,6 +103,30 @@
       "name": "other_party_last_name",
       "value": "Butler"
     },
+    {
+      "name": "is_other_party_address_known_yes",
```

Comment:
Where have you got this field from? I can't see it in the SIP. It's called 'is_other_party_address_known'...

Reply:
Logic changed after change in requirement